src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReportingServiceTest.java

```diff
@@ -93,12 +101,46 @@ void should_save_task_and_get_task_from_replica_tables() {
                     assertEquals(1, taskHistoryResourceList.size());
                     assertEquals(taskResource.getTaskId(), taskHistoryResourceList.get(0).getTaskId());
                     assertEquals(taskResource.getTaskName(), taskHistoryResourceList.get(0).getTaskName());
+                    assertEquals(taskResource.getTitle(), taskHistoryResourceList.get(0).getTaskTitle());
 
                     return true;
                 });
     }
 
+    @Test
+    void should_save_task_and_get_task_from_reportable_task() {
+        TaskResource taskResource = createAndSaveTask();
+
+        await().ignoreException(AssertionFailedError.class)
+            .pollInterval(1, SECONDS)
+            .atMost(10, SECONDS)
+            .until(
+                () -> {
+                    List<ReportableTaskResource> reportableTaskList
+                        = miReportingService.findByReportingTaskId(taskResource.getTaskId());
+
+                    assertFalse(reportableTaskList.isEmpty());
+                    assertEquals(1, reportableTaskList.size());
+                    assertEquals(taskResource.getTaskId(), reportableTaskList.get(0).getTaskId());
+                    assertEquals(taskResource.getTaskName(), reportableTaskList.get(0).getTaskName());
+                    assertEquals(taskResource.getTitle(), reportableTaskList.get(0).getTaskTitle());
 
+                    return true;
+                });
+    }
```

Comment:
Can we add the AC for Assigned, AutoAssign without reconfiguration? I.e. AC1 ->  A/C1: (Insert a new record based a new task being created and its auto assigned to a user)

Reply:
Added new test to cover this AutoAssign